docs/Installation.md

```diff
@@ -16,12 +16,8 @@ Build Support_ component when installing Unity.
        width="500" border="10" />
 </p>
 
-## Windows Users
-For setting up your environment on Windows, we have created a [detailed
-guide](Installation-Windows.md) to setting up your env. For Mac and Linux,
-continue with this guide.
-
-## Mac and Unix Users
+## Environment Setup
+For setting up your environment follow this [guide](Using-Virtual-Environment.md).
```

Comment:
Maybe re-write to something like: "We now support a single mechanism for installing ML-Agents on Mac/Windows/Linux using Virtual Environments. For more information on Virtual Environments and installation instructions, follow this [guide](Using-Virtual-Environment.md)." 

I think this would be a more colorful message, esp for earlier users of ML-Agents.

Reply:
done